05/Task05.js

```diff
@@ -1,10 +1,54 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react'
+import Slider from './../src/components/slider/Slider'
+import { Switch, Route, useHistory } from 'react-router-dom'
+import Shop from '../src/components/Shop'
+import products from './../src/products.json'
+import FilterNameForm from '../src/components/FilterNameForm'
+import FilteredProducts from '../src/components/FilteredProducts'
 
 const Task05 = () => {
-    return (
-        <h1>Task05</h1>
-    );
-}
+	const history = useHistory()
+	const [sliderValue, setSliderValue] = useState([])
+	const [submittedSliderValue, setSubmittedSliderValue] = useState([])
+	const [inputValue, setInputValue] = useState('')
+	const [submittedInputValue, setSubmittedInputValue] = useState('')
+
+	const handleSliderChange = value => {
+		setSliderValue(value)
+	}
+
+	const handleInputChange = e => {
+		setInputValue(e.target.value)
+	}
+
+	const handleSubmit = e => {
+		e.preventDefault()
+		setSubmittedSliderValue(sliderValue)
+		setSubmittedInputValue(inputValue)
+		history.push(`/task05/minPrice${sliderValue[0]},maxPrice${sliderValue[1]},search="${submittedInputValue}"`)
```

Comment:
Zobacz komentarz do linii 47, teraz trzeba osadzić dane zgodnie z regułami w tamtej linii tj.
```history.push(`/task05/${sliderValue[0]}-${sliderValue[1]},${submittedInputValue}`)```

Uwaga! Coś to nie tak jest z przekazywaniem wartości z pola input! Trzeba zmienić z `submittedInputValue` na `inputValue`